src/angular/1-why-angular/why-angular.md

```diff
@@ -13,22 +13,22 @@ Some of the best things about Angular are that it is opinionated, has built-in t
 
 ### 1. An Opinionated Framework
 
-Developers have to make a lot of decisions on a daily basis, which can often create decision fatigue. The great part about using an opinionated framework is being able to shift your focus from build configs, high level architecture decisions,   to writing the actual client-specific functioning pieces of the application.
+Developers have to make a lot of decisions on a daily basis, which can often create decision fatigue. The great part about using an opinionated framework like Angular, is being able to shift your focus from build configs and high level architecture decisions, to writing the actual client-specific functioning pieces of the application. 
 
-The use of TypeScript to force type checking improves workflow by catching errors at compile time allowing teams to catch potential errors much faster.
+The use of Typescript to force type checking improves developer workflow by catching errors at compile time, allowing teams to catch potential errors much faster.
```

Comment:
correct typescript casing is TypeScript, not Typescript

Reply:
https://www.typescriptlang.org/ <-- yea you're right lol. Gonna correct that tyty

Reply:
corrected~